test/integration/pending_loans_integration_test.rb

```diff
@@ -3,16 +3,6 @@
 class PendingLoansIntegrationTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
 
-  test "a cart starts empty" do
```

Comment:
There's no button for them to click to view their cart.  We could test if the session[:pending_loan] is empty, but that doesn't seem like a good test.